includes/content-distribution/class-incoming-post.php

```diff
@@ -584,9 +607,9 @@ public function insert( $payload = [] ) {
 		/**
 		 * Fires after an incoming post is inserted.
 		 *
-		 * @param int   $post_id   The post ID.
-		 * @param bool  $is_linked Whether the post is linked.
-		 * @param array $payload   The post payload.
+		 * @param int $post_id The post ID.
+		 * @param bool $is_linked Whether the post is linked.
+		 * @param array $payload The post payload.
```

Comment:
Why this change in linting?

Reply:
It was probably just my editor when it autoformats. There are too many spaces I think it picked up?

Reply:
It follows the standard we use for functions and methods so the parameters info is aligned. It could be that your IDE doesn't interpret the `do_action()` as a thing that can precede that type of docblock.

Reply:
You are right! I had the editor set to not WP for some reason and it felt like I had to keep tweaking it :)